src/main/java/org/apache/fineract/operations/TransactionRequest.java

```diff
@@ -20,7 +20,7 @@
 public class TransactionRequest extends AbstractPersistableCustom<Long> {
 
     @Column(name = "WORKFLOW_INSTANCE_KEY")
-    private String workflowInstanceKey;
+    private Long workflowInstanceKey;
```

Comment:
Can we avoid this? I think you're aware of the +/-3 parsing issues when process instance keys are in Long format.

Joins should be possible with Strings as well.

Reply:
In `Variables` DTO, `workflowInstanceKey` is Long but in `TransactionRequest` DTO, `workflowInstanceKey` is String. So join comparison is giving compile time error.

Reply:
@danishjamal104 : Let's make it String everywhere to avoid issues related with Long.

Reply:
done